.github/workflows/publish-release.yml

```diff
@@ -20,7 +20,10 @@ jobs:
               env:
                 packageUser: ${{ github.actor }}
                 packagePAT: ${{ secrets.GITHUB_TOKEN }}
-              run: ./gradlew build -x check -x test
+              run: |
+                git config --global user.name ${{ secrets.BALLERINA_BOT_USERNAME }}
+                git config --global user.email ${{ secrets.BALLERINA_BOT_EMAIL }}
+                ./gradlew build -x check -x test -x :graphql-examples:build
```

Comment:
You have to skip module-specific examples. `-x :graphql-examples:build` will fail the build. Plus, if the module do not have `check` task, this will fail.

Reply:
Fixed in 535a72f